File: problem_solving_for_beginners/string_problems/python/count_vowel.py
#!/usr/bin/python3
"""Count vowels in a string"""

str1 = input("Enter a string: ")
length = 0

for i in str1:
	length += 1
#write your code here
vowels = ['a', 'e', 'i', 'o', 'u']
count = 0

for ch in str1:
    if ch == 'a' or ch == 'e' or ch == 'i' or ch == 'o' or ch == 'u':
            count += 1

print(count)

File: problem_solving_for_beginners/string_problems/c-programming/count_vowel.c
#include<stdio.h>
/**
 * main - Count vowels in a string
 *
 * Return: 0 Always (success)
 */
int main(void)
{
	char str[10];
	printf("Enter a string: ");
	scanf("%s", str);

	//Write your code here
	char vowels[] = {'a', 'e', 'i', 'o', 'u', '\0'};
	int count = 0, i, j;
	
	for (i = 0; str[i] != '\0'; i++)
	{
	    for (j = 0; vowels[j] != '\0'; j++)
	    {
	        if (str[i] == vowels[j])
	            count++;
	    }
	}
    printf("%d\n", count);
	return 0;
}